app/models/user.rb

```diff
@@ -11,7 +11,8 @@ class User < ApplicationRecord
     length: {maximum: Settings.user.email.max_length},
     format: {with: VALID_EMAIL_REGEX},
     uniqueness: {case_sensitive: false}
-  validates :password, length: {minimum: Settings.user.password.min_length}
+  validates :password, length: {minimum: Settings.user.password.min_length},
+    allow_nil: true
```

Comment:
Dạ thưa chị, trong trường hợp cập nhật thông tin, nếu như không muốn đổi mật khẩu thì phải dùng `allow_nil: true` để tránh báo lỗi `password can't be blank` (dùng trong `has_secure_password`) ở validation ạ (mật khẩu cũ thỏa mãn validation từ lúc đăng ký nên không cần kiểm tra lại)